### ISSUE TITLE
Add support for F-droid OTA zip?

### ISSUE THREAD

**Issue by Brli** [casualsnek/waydroid_script#11]
F-Droid comes in handy as alternative to gapps store, and is mentioned in waydroid's doc too

is it possible to add it to the script for easier installation of F-droid app?

Thank you.

https://f-droid.org/en_US/packages/org.fdroid.fdroid.privileged.ota/

**Comment by casualsnek** [comment]
> F-Droid comes in handy as alternative to gapps store, and is mentioned in waydroid's doc too
> 
> is it possible to add it to the script for easier installation of F-droid app?
> 
> Thank you.
> 
> https://f-droid.org/en/packages/org.fdroid.fdroid.privileged.ota/

yes, it should be possible to do this, maybe i will add it in next update ;)
